json-endpoints/jetpack/class.jetpack-json-api-themes-delete-endpoint.php

```diff
@@ -21,14 +21,18 @@ protected function delete() {
 				continue;
 			}
 
-			$result = delete_theme( $theme );
+			// Allow alternative theme deletion mechanism.
+			$result = apply_filters( 'jetpack_wpcom_theme_delete', false, $theme );
```

Comment:
Could you add a docblock for this new filter? See [this](https://github.com/Automattic/jetpack/pulls?q=is%3Apr+label%3ADocs+is%3Aclosed) for some examples.